data/notifications/x-nemo.call.missed.preview.conf

```diff
@@ -0,0 +1,6 @@
+appIcon=icon-lock-missed-call
+transient=true
+x-nemo-preview-icon=icon-lock-missed-call
+x-nemo-feedback=call
+x-nemo-priority=120
+x-nemo-led-disabled-without-body-and-summary=false
```

Comment:
I assume these two notifications don't need x-nemo-display-on because the display should have been turned on for the call already?